### DISCUSSION TITLE
Reeksen kopiëren van de ene cursus naar de andere?

### DISCUSSION THREAD

**Discussion by jonathanvynck** [dodona-edu/feedback-and-support#44] [Feature requests]
Is dit mogelijk en zoja, hoe?

**Comment by bmesuere** [comment]
Een reeks integraal kopiëren is niet mogelijk. Het is wel mogelijk om een volledige cursus (inclusief reeksen) te kopiëren, of om de reeks opnieuw samen te stellen met dezelfde oefeningen.

**Comment by Klaasvdb** [comment]
Staat deze feature nog op de Todo-lijst? Ik zou hier hard mee gebaat zijn.

**Reply by jorg-vr** [reply]
Deze feature staat momenteel niet op onze TODO-lijst.
Zie je het eventueel zitten om in iets meer detail te beschrijven waarvoor je deze feature zou willen gebruiken?

Dit helpt ons het belang van de feature in te schatten, maar zorgt er ook voor dat we hier bij de uiteindelijke implementatie rekening mee kunnen houden, zodat de nieuwe feature ook echt jouw huidige problemen oplost.

**Reply by IngeVerbeek** [reply]
Het zou handig zijn wanneer je de cursus aan meerdere klassen geeft, maar op andere tijdstippen. Daardoor liggen de deadlines ook anders (zowel voor het indienen van oefeningenreeksen als voor toetsen). 
Een alternatief kan misschien zijn dat deadlines voor een reeks kunnen aangepast worden per (groep van) studenten?

**Comment by bmesuere** [comment]
Dag @Klaasvdb, @jonathanvynck en @IngeVerbeek 

Deze functie werd recent toegevoegd aan Dodona. Meer informatie kan je vinden op https://github.com/orgs/dodona-edu/discussions/157
